lib/devices/android/selendroid.js

```diff
@@ -220,32 +220,24 @@ Selendroid.prototype.checkInternetPermissionForApp = function (cb) {
 };
 
 Selendroid.prototype.checkSelendroidCerts = function (cb) {
-  var alreadyReturned = false
-    , checks = 0;
-
-  var onDoneSigning = function () {
-    checks++;
-    if (checks === 2 && !alreadyReturned) {
-      cb();
-    }
-  };
-
-  // these run in parallel
   var apks = [this.selendroidServerPath, this.args.app];
-  _.each(apks, function (apk) {
+    
+  async.eachSeries(apks, function(apk, done){
```

Comment:
It would be readable if you can wrap iterator function in a var and pass it to async.series